### PR TITLE
clang-tidy applied to tests

### PR DESCRIPTION
**Pull request by neheb**

### CODE REVIEW

File: test/config/test_configgenerator.cc [hunk 1]
```diff
@@ -11,10 +11,10 @@ using namespace ::testing;
 class ConfigGeneratorTest : public ::testing::Test {
 
 public:
-    ConfigGeneratorTest() {};
-    virtual ~ConfigGeneratorTest() {};
+    ConfigGeneratorTest() = default;;
```
Comment (KarlStraussberger) [inline test/config/test_configgenerator.cc:14]: I suppose the second `;` is unwanted and happened by replacing the braces with `default`.

**Review by KarlStraussberger — CHANGES_REQUESTED** [review]

File: test/config/test_configgenerator.cc (hunk 1 above)
Comment (neheb) [reply test/config/test_configgenerator.cc:14]: Sounds about right. The ; was originally not necessary anyway.

**Comment by neheb** [comment]
funny. freebsd build is stuck.

**Comment by KarlStraussberger** [comment]
> funny. freebsd build is stuck.

Sourcehut seems to have stopped sending responses to github

**Review by KarlStraussberger — APPROVED** [review]

**Pull request merged**